storage/local/chunk.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/prometheus/prometheus/storage/metric"
 )
 
+// The DefaultChunkEncoding can be changed via a flag.
```

Comment:
First of all, this is obviously totally not related to this PR. I only added the doc comment here because somebody (I guess a @fabxc ) forgot to add it previously.

We had that discussion when we introduced the flag. (Single) delta encoding is not "virtually no compression". It could even work better than double-delta encoding, given the right data set. The reason to have a flag is to create backwards-compatible series files if desired. Also, if data is completely random, there will be no compression anyway, but double-delta encoding is slower than delta encoding.